studio/components/interfaces/TableGridEditor/SidePanelEditor/TableEditor/ColumnManagement.tsx

```diff
@@ -318,8 +318,8 @@ const ColumnManagement: FC<Props> = ({
               Add column
             </Button>
           )}
-          <Link href="https://supabase.com/docs/guides/database/tables#data-types">
-            <a>
+          <Link href="https://supabase.com/docs/guides/database/tables#data-types" passHref>
+            <a target="_blank" rel="noopener noreferrer">
```

Comment:
suggestion to remove <a> tag and use `as`  prop in button so that it renders as an `<a/>` tag

```suggestion
<Button
  as="a"
  target="_blank" 
  rel="noopener noreferrer"
  type="text"
  className="text-scale-1000 hover:text-scale-1200"
  icon={<IconExternalLink size={12} strokeWidth={2} />}
>
  Learn more about data types
</Button>
```